### ISSUE TITLE
Add Google Drive data source

### ISSUE THREAD

**Issue by samber** [mholt/timeliner#11]
With google file conversion (such as gdoc => docx).

**Comment by mholt** [comment]
This needs a lot of discussion, because I can already imagine a dozen or so ways this would work...

**Comment by alexrdzs** [comment]
Yes!  This is exactly the workflow I'm looking for (file conversion). So far I've been doing it manually by downloading a complete folder, since all of its contents get converted to docx, etc.

Then I'move them to an external HD ready to be backed up, and used in the future without the need of Google Docs.

**Comment by mholt** [comment]
@samber FYI, I've added [a new issue template](https://raw.githubusercontent.com/mholt/timeliner/master/.github/ISSUE_TEMPLATE/data_source.md) for requesting data sources which will make it easier for whoever implements it. If you could update your original post to use [this template](https://raw.githubusercontent.com/mholt/timeliner/master/.github/ISSUE_TEMPLATE/data_source.md), that would be great, thanks!
